src/main/tut/README.md

```diff
@@ -226,3 +226,10 @@ Run `sbt test` to build the project and exercise the unit test suite.
 Rubiz is versioned with [semver](http://semver.org/) and released to [sonatype](https://oss.sonatype.org/).
 Open an issue on Github if you feel there should be a release published that hasn't been. You can run `sbt tut`
 to regenerate documentation locally if you modify the docs in the `/main/tut` directory.
+
+For those with permission to release:
+* Install `gpg` and [generate a key](https://www.gnupg.org/gph/en/manual.html#AEN26). Upload that key to a [public key server](http://pgp.mit.edu/).
+    * Mac users can `brew install gpg pinentry-mac` to get the tools needed.
+* Create a Sonatype [credentials file](http://www.scala-sbt.org/1.0/docs/Using-Sonatype.html#Fourth+-+Adding+credentials).
+* Run `sbt release`
+* `git push` to add the new tags and release commits to master.
```

Comment:
Your assumption is that all users that can publish don't have their own forks, which isn't the way people should do development on OSS projects (IMO). Maybe you shouldn't give a command and instead say they should push the tags and version bumps to the `rubicon-project/rubiz`